pkg/webhook/resources/setting/validator.go

```diff
@@ -1465,6 +1545,25 @@ func (v *settingValidator) checkStorageNetworkRangeValid(config *storagenetworkc
 	return nil
 }
 
+func (v *settingValidator) checkVMMigrationNetworkRangeValid(config *networkutil.Config) error {
+	nodes, err := v.nodeCache.List(labels.Everything())
+	if err != nil {
+		return werror.NewInternalError(err.Error())
+	}
+
+	count, err := webhookUtil.GetUsableIPAddressesCount(config.Range, config.Exclude)
+	if err != nil {
+		return err
+	}
+
+	// 1 node has 1 virt-handler which needs 1 IP address.
+	if count < len(nodes) {
+		return fmt.Errorf("allocatable IP address range is < %d,allocate sufficient range", len(nodes))
+	}
+
```

Comment:
should we have a check to ensure no VM migrations are in flight before accepting the change to the setting?